torch/quantization/_numeric_suite.py

```diff
@@ -115,6 +118,22 @@ def forward(self, x, y):
             self.stats["float"] = torch.cat((self.stats["float"], y.detach()))
 
 
+class TensorLogger(Logger):
```

Comment:
Can we reuse Record Observer here?

Reply:
Record Observer inherits from observer, since we want to unify the shadow comparison and output comparison so that they can share same functions such as get_observer_dict() we need to inherit both from the same base Logger class.